packages/ui/src/components/shadcn/ui/input.tsx

```diff
@@ -1,5 +1,5 @@
 import { VariantProps, cva } from 'class-variance-authority'
```

Comment:
```suggestion
'use client'

import { VariantProps, cva } from 'class-variance-authority'
```

needed to not fail the design-system and docs builds because app router